calculator/mod_list.go

```diff
@@ -42,7 +41,8 @@ func (m *ModList) AddDB(db *ModList) {
 	if db == nil {
 		return
 	}
-	m.mods = utils.CopySlice(m.mods)
```

Comment:
Might be because it would create a memory leak? According to this article a slice is just pointing to allocated memory so you need to copy the slice to avoid keeping the original allocations in memory. https://go.dev/blog/slices-intro

I'll update it to copy the `db.mods`.